codecov.yml

```diff
@@ -0,0 +1,22 @@
+# Customizations to codecov for Trillian repo. This will be merged into
+# the team / default codecov yaml file.
+#
+# Validate changes with:
+# curl --data-binary @codecov.yml https://codecov.io/validate
+
+# Exclude code that's for testing, demos or utilities that aren't really
+# part of production releases.
+ignore:
+  - "cmd/createtree/testonly"
+  - "crypto/keys/testonly"
+  - "docs/**/*"
+  - "examples/**/*"
+  - "integration/**/*"
+  - "monitoring/testonly"
+  - "server/mock_log_operation.go"
```

Comment:
Could we not use a pattern like "mock_*.go" to match all mocks?

Reply:
I'm not sure the recursive matching is smart enough. It might be but the docs weren't entirely clear.

Reply:
Maybe. I can try it and see if it excludes the right things.

Reply:
We can test it by looking at the coverage report, can't we?